cldf/languages.csv

```diff
@@ -1,68 +1,68 @@
 ID,Name,Glottocode,Glottolog_Name,ISO639P3code,Macroarea,Latitude,Longitude,Family
```

Comment:
any idea why the lat/long/family are being lost?

Reply:
No! I didn't touch the code there and it is working for other datasets... I'll try to fix it.